src/main/java/org/disq_bio/disq/impl/file/TbiMerger.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Disq
+ *
+ * MIT License
+ *
+ * Copyright (c) 2018 Disq contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.disq_bio.disq.impl.file;
+
+import htsjdk.samtools.seekablestream.SeekableStream;
+import htsjdk.tribble.index.tabix.TabixIndexMerger;
+import htsjdk.tribble.util.TabixUtils;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.apache.hadoop.conf.Configuration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** Merges tabix index files for (headerless) parts of a VCF file into a single index file. */
+public class TbiMerger {
```

Comment:
In gatk we're going to want a TbiMerger that works on tbi's from files that have headers and footers.  I'm not sure if we should anticipate a naming conflict and call this something like "TbiMergerForPartsFiles" or just deal with that issue in gatk later.